compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -99,6 +100,13 @@ void compute_asym_scale_zp(float min, float max, float &scaling_factor, int64_t
     nudged_zero_point = static_cast<uint8_t>(std::round(zero_point_double));
   }
 
+  // protect scale from being very low due to overflow
+  if (scale < 1e-5)
```

Comment:
Just curiosity. How did you get the number (1e-5) ? Is it selected empirically?

Reply:
yes. When we tested by calculating the scale factor range that does not overflow, it was empirically **1e-5**.